main/NetworkImage/index.tsx

```diff
@@ -26,11 +32,11 @@ type Styles = {
 interface Props {
   style?: StyleProp<ViewStyle>;
   styles?: Styles;
-  url: string | undefined;
+  url: string;
```

Comment:
I'm curious why this type is changed. Can you explain why?

Reply:
Url is essential props on `Image` like component. So I thought no need to be `undefined` or `null`. But I thought again in the morning it would be better to put `undefined`. 😅  I will fix it right now.